packages/www/src/helpers/monaco-extra-libs/index.ts

```diff
@@ -24,14 +24,14 @@ export default function getMonacoEditorExtraLibs() {
     const route = brisaDep.match(pathname);
     const filePath = route.filePath.replace(root, '');
     const fileRaw = fs.readFileSync(route.filePath, 'utf-8');
-    extraLibs += `monaco.languages.typescript.typescriptDefaults.addExtraLib(\`${fileRaw}\`, 'file://${filePath}');`;
+    extraLibs += `monaco.languages.typescript.typescriptDefaults.addExtraLib(${JSON.stringify(fileRaw)}, 'file://${filePath}');`;
```

Comment:
After the change to fileSystemRouter, it does import the .d.ts files (although Monaco Editor does not like it yet), but now it is necessary to stringify so that it does not break the playground, since inside the content of these files there are quotation marks and things that otherwise escape.